packages/block-library/src/social-links/block.json

```diff
@@ -6,6 +6,9 @@
 		"openInNewTab": {
 			"type": "boolean",
 			"default": false
+		},
+		"iconSize": {
```

Comment:
Does this attribute need the icon prefix or could it be just "size"?

Reply:
Switched by Joen, thanks Joen!